flow-client/src/test/frontend/AuthenticationTests.ts

```diff
@@ -71,13 +71,11 @@ describe('Authentication', () => {
     it('should return a CSRF token on valid credentials', async () => {
       fetchMock.post('/login', {
         body: happyCaseResponseText,
-        redirectUrl: '/'
+        redirectUrl: 'localhost:8080/'
```

Comment:
Incorrect URL, `localhost:` could be recognised as a protocol (http?). Although likely not an issue for this test, let’s change to a correct form.

```suggestion
        redirectUrl: '//localhost:8080/'
```